### PR TITLE
Feature/auto subscription notification

### PR DESCRIPTION
**Pull request by Lex-23**
Write logic for two endpoints:
1)POST:  "/api/tg_notify-on" - get data from tg_bot and fill chat_id to consumer.data (it related consumer with tg service)
2)POST: "/api/notifications/create_consumer/<str:consumer_type>/" - created consumer, and created data for notification service if need

### CODE REVIEW

**Comment by Lex-23** [comment]
[Auto subscription for notification for user](https://trello.com/c/OlDZgYab/42-auto-subscription-for-notification-for-user)

File: cryptostock_api/cryptostock/notification/urls.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+from notification.views import (
+    CreateConsumerApiView,
+    TelegramNotificationActivateApiView,
+)
+
+urlpatterns = [
+    path("tg_notify-on", TelegramNotificationActivateApiView.as_view()),
+    path(
+        "notifications/create_consumer/<str:consumer_type>/",
```
Comment (vadim-denisov) [inline cryptostock_api/cryptostock/notification/urls.py:10]:
```suggestion
        "notifications/consumers/<str:consumer_type>/",
```

File: cryptostock_api/cryptostock/utils/validators.py [hunk 2]
```diff
@@ -64,3 +65,13 @@ def get_validated_asset_from_market(asset_name, market):
             [f"asset {asset_name} not allow for market {market.name}."]
         )
     return asset
+
+
+def validate_consumer_type(consumer_type: str):
+    if consumer_type not in ConsumerType.__members__:
```
Comment (vadim-denisov) [inline cryptostock_api/cryptostock/utils/validators.py:71]: does it work?
```suggestion
    if consumer_type not in ConsumerType:
```

File: cryptostock_api/cryptostock/notification/urls.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+from notification.views import (
+    CreateConsumerApiView,
+    TelegramNotificationActivateApiView,
+)
+
+urlpatterns = [
+    path("tg_notify-on", TelegramNotificationActivateApiView.as_view()),
```
Comment (vadim-denisov) [inline cryptostock_api/cryptostock/notification/urls.py:8]:
```suggestion
    path("notifications/consumers/<str:consumer_type>/activate", ...),
```

File: cryptostock_api/cryptostock/utils/validators.py (hunk 2 above)
Comment (Lex-23) [reply cryptostock_api/cryptostock/utils/validators.py:71]: Not, it doesn't

Comment (Lex-23) [reply cryptostock_api/cryptostock/utils/validators.py:71]: TypeError: unsupported operand type(s) for 'in': 'str' and 'EnumMeta'

File: cryptostock_api/cryptostock/notification/urls.py (hunk 1 above)
Comment (Lex-23) [reply cryptostock_api/cryptostock/notification/urls.py:10]: got it

File: cryptostock_api/cryptostock/notification/urls.py (hunk 3 above)
Comment (Lex-23) [reply cryptostock_api/cryptostock/notification/urls.py:8]: got it

File: cryptostock_api/cryptostock/utils/validators.py (hunk 2 above)
Comment (andreyPromaster) [reply cryptostock_api/cryptostock/utils/validators.py:71]: I think that better enumeration types provided by djangio https://docs.djangoproject.com/en/4.0/ref/models/fields/#enumeration-types

File: cryptostock_notification_services/telegram_bot/main.py [hunk 4]
```diff
@@ -12,17 +13,36 @@
...
-    )
+    arguments = message.get_args()
+    if arguments:
+        chat_id = message.from_user.id
+        payload = {
+            "source": TELEGRAM_BOT_NAME,
+            "account_token": arguments,
+            "chat_id": chat_id,
+        }
+        requests.post(TELEGRAM_NOTIFICATION_ACTIVATE_URL, data=payload)
```
Comment (andreyPromaster) [inline cryptostock_notification_services/telegram_bot/main.py:33]: I suppose that should check response status and log it if response not success

Comment (andreyPromaster) [reply cryptostock_notification_services/telegram_bot/main.py:33]: And in my opinion need notify user if provided data not valid

Comment (Lex-23) [reply cryptostock_notification_services/telegram_bot/main.py:33]: I think too and changed this

**Review by vadim-denisov — APPROVED** [review]

**Pull request merged**